packages/prosemirror-lwdita/src/github-integration/request.ts

```diff
@@ -32,7 +32,7 @@ export interface URLParams {
  * source = GitHub resource,
  * referer = Referer of the request
  */
-export const validKeys = ['ghrepo', 'source', 'referer'];
+export const validKeys = ['ghrepo', 'source', 'branch', 'referer'];
```

Comment:
tsdoc needs updating.

Reply:
Fixed in b6962288f2f38f863a94da06d3e6f53b77fe42fa .